public/app/features/alerting/unified/components/rule-viewer/Actions.tsx

```diff
@@ -71,6 +73,19 @@ export const useAlertRulePageActions = ({ handleDelete, handleDuplicateRule }: P
               childItems={[<ExportMenuItem key="export-with-modifications" identifier={identifier} />]}
             />
           )}
+          {rulePluginLinkExtension.length > 0 && (
```

Comment:
I would move this piece of code to a separate component

Reply:
This code only renders additional `<Menu.Item>` components as the code below and above it
Wouldn't it be more confusing if we render another component in the middle of `<Menu.Item>` elements?